pylint/lint/expand_modules.py

```diff
@@ -46,6 +46,20 @@ def _is_in_ignore_list_re(element: str, ignore_list_re: list[Pattern[str]]) -> b
     return any(file_pattern.match(element) for file_pattern in ignore_list_re)
 
 
+def is_ignored_file(
```

Comment:
```suggestion
def _is_ignored_file(
```

Let's make this private.